### DISCUSSION TITLE
Rethink the icons and links under maintainers on README.md file

### DISCUSSION THREAD

**Discussion by y4izus** [Qiskit/qiskit.org#2371] [Ideas]
The icons we are using are:

| Icon | Label | Points to |
|-----|-------|----------|
| 🐛 | Bug reports | Issues created by the maintainer |
| 💬 | Answering questions on slack | Get invited to slack link |
| 💻 | Code | Commits of the maintainer |
| 🎨 | Design | Issues created by Russell |

IMO, we should rethink some labels and links, for example:
- Bug report is related to the issues labeled with `bug`, not with all the issues
- All the chat links point to the same URL (that changes over time). If we want to be able to the people to chat, maybe we should use the same approach we have already used for advocates

What do you think?

**Comment by eddybrando** [comment]
I like the idea of rethinking this labels. 🙂

I believe that we should start by thinking from a user's POV.

In this case, the person that is most likely to reach the **Readme** file and its **Maintainers** section, is a developer interested in our project. It may be that they want to know who to contact in order to solve a particular issue, or maybe they just want general information like the size of the core team.

I find the **🎨 Design** label and link super helpful when wanting to quickly find the design issues. The other **💻 Code** links maybe less so, because I don't think that it's a common scenario to be interested in seeing the commits from X or Y maintainer. But it doesn't bug me either. It's a neat feature. 😬

The **🐛 Bug reports** links is for me similar to the **💻 Code** ones. I don't see a real need, but I find them neat. The only think lacking here IMO would be to either add another label to display the non-bug issues, or to change it to a label to link to all issues. I like the former, but I can't think of an icon of wording that wouldn't logically conflict with the fact that _Bugs_ are also a kind of _Issue_.

The **💬 Answering questions on slack** link is IMO a little bit deceiving. I personally would expect to be linked to the direct messages of that user instead of the general Slack page. I like @y4izus's proposal about this.
